src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA&#47A)">
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        class="card__image"
        alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA&#47A)"
      >
```
closing bracket should be on the same level as opening one

Reply:
+